src/templates/base.njk

```diff
@@ -87,14 +87,13 @@
               <span
                 class="bg-gray-800 text-white px-3 py-2 rounded-md text-sm font-medium mr-2" id="movrPrice"
                 x-init="
-
-                  response = await fetch('https://movr-price-c17w.vercel.app/api/movr', {
+                  response = await fetch('https://moonsama-serverside-priceticker.vercel.app/api/movrprice', {
```

Comment:
Price ticker renders with correct price data
![image](https://user-images.githubusercontent.com/47224690/208292970-5c7e0f3f-e505-4dd0-9e41-284e7b3c007e.png)

Reply:
Total price in usd (derived from movr price) correct 
![image](https://user-images.githubusercontent.com/47224690/208293174-bb1e2de1-76a2-44cd-b3ac-05e7f4c963ce.png)